packages/ui-components/src/Card/TableCardRow.js

```diff
@@ -12,6 +12,7 @@ export default function TableCardRow ({
   badgeColor = 'secondary',
   tooltip = null,
   onClickBadge = () => {},
+  loadIcon,
```

Comment:
`loading` will better than `loadIcon`,  `loading` indicates the state, while the `loadIcon` indicates the component